components/rmrk/Create/SimpleMint.vue

```diff
@@ -506,6 +506,7 @@ export default class SimpleMint extends mixins(
     this.isLoading = true
     this.status = 'loader.ipfs'
     const { accountId, version } = this
+    this.rmrkMint.max = Number(this.rmrkMint.max)
```

Comment:
Why convert its type two times in this file? Could we only keep one of them?

Reply:
done